### PR TITLE
added EVM module general and event section

### PR DESCRIPTION
**Pull request by web3dopamine**

### CODE REVIEW

File: versioned_docs/version-2.0/modules/evm/index.md [hunk 1]
```diff
@@ -13,25 +13,224 @@ Moralis.Evm.xxx()
...
+_provider: Proxy {_events: {…}, _eventsCount: 4, _maxListeners: 100, _log: u, _state: {…}, …}
+```
+
+
 #### provider
+
+Get info on connection provider
+
+```js
+Moralis.Evm.connector()
```
Comment (abdulmalik97) [inline versioned_docs/version-2.0/modules/evm/index.md]: Should be provider here

File: versioned_docs/version-2.0/modules/evm/index.md [hunk 2]
```diff
@@ -13,25 +13,224 @@ Moralis.Evm.xxx()
...
+lowercase: (...)
+[[Prototype]]: Object
+```
+
 #### isConnected
+
+Check whether web3 provider is connected or not
+
+```js
+Moralis.Evm.isConnected()
```
Comment (abdulmalik97) [inline versioned_docs/version-2.0/modules/evm/index.md]: Can we add output log here too?

File: versioned_docs/version-2.0/modules/evm/index.md [hunk 3]
```diff
@@ -44,3 +243,18 @@ Moralis.Evm.xxx()
 #### transferErc721
 #### transferErc1155
 #### executeFunction
+
+
+
+
+Send request to moralis server
```
Comment (abdulmalik97) [inline versioned_docs/version-2.0/modules/evm/index.md]: What is this description for?

File: versioned_docs/version-2.0/modules/evm/index.md (hunk 2 above)
Comment (web3dopamine) [reply versioned_docs/version-2.0/modules/evm/index.md]: Not sure how, its not in the demo test app

File: versioned_docs/version-2.0/modules/evm/index.md (hunk 3 above)
Comment (web3dopamine) [reply versioned_docs/version-2.0/modules/evm/index.md]: Removed

**Pull request closed without merging**